datasets/leidingeninfrastructuur/dataset.json

```
@@ -14,38 +14,42 @@
   "accrualPeriodicity": "dagelijks",
   "spatialDescription": "Gemeente Amsterdam",
   "theme": [
-    "leidingeninfrastructuur",
+    "ondergrondse infrastructuur",
     "Ruimte en Topografie"
   ],
-  "owner": "Gemeente Amsterdam, Ruimte & Economie",
+  "owner": "Gemeente Amsterdam",
   "auth": "OPENBAAR",
-  "authorizationGrantor": "datateam omgevingswet",
+  "authorizationGrantor": "datateamstedelijkeonwikkelingenbeheer@amsterdam.nl",
   "keywords": [
     "leidingen",
-    "infrastuctuur"
+    "infrastuctuur",
+    "klic",
+    "waternet",
+    "rioolnetwerk",
+    "liander"
   ],
   "hasBeginning": "2021-11-01T00:00:00+01:00",
   "crs": "EPSG:28992",
   "tables": [
     {
-      "id": "liggingLijnTotaal",
-      "$ref": "liggingLijnTotaal/v1.2.0",
+      "id": "kadasterKlicKabelsEnLeidingen",
+      "$ref": "kadasterKlicKabelsEnLeidingen/v1.0.0",
       "activeVersions": {
-        "1.2.0": "liggingLijnTotaal/v1.2.0"
+        "1.0.0": "kadasterKlicKabelsEnLeidingen/v1.0.0"
       }
     },
     {
-      "id": "liggingPuntTotaal",
-      "$ref": "liggingPuntTotaal/v1.2.0",
+      "id": "kadasterKlicPuntobjecten",
+      "$ref": "kadasterKlicPuntobjecten/v1.0.0",
       "activeVersions": {
-        "1.2.0": "liggingPuntTotaal/v1.2.0"
+        "1.0.0": "kadasterKlicPuntobjecten/v1.0.0"
       }
     },
     {
-      "id": "liggingVlakTotaal",
-      "$ref": "liggingVlakTotaal/v1.2.0",
+      "id": "kadasterKlicEisvoorzorgsmaatregelen",
```

Deze is nog meer te lang.

```suggestion
      "id": "kadasterKlicEisvoorzorgsmaatregelen",
      "shortname": "klicEisvoorzorg",
```